python_modules/dagster-graphql/dagster_graphql/schema/schedules/schedules.py

```diff
@@ -151,28 +152,29 @@ def resolve_futureTick(self, _graphene_info: ResolveInfo, tick_timestamp: int):
         )
 
     def resolve_potentialTickTimestamps(
-        self, _graphene_info: ResolveInfo, **kwargs: Dict[str, object]
+        self,
+        _graphene_info: ResolveInfo,
+        start_timestamp: Optional[float] = None,
+        upper_limit: Optional[int] = None,
+        lower_limit: Optional[int] = None,
     ):
         """Get timestamps when ticks will occur before and after a given timestamp.
 
         upper_limit defines how many ticks will be retrieved after the current timestamp, and lower_limit defines how many ticks will be retrieved before the current timestamp.
         """
-        start_timestamp = cast(
-            float,
-            kwargs.get(
-                "start_timestamp", get_timestamp_from_utc_datetime(get_current_datetime_in_utc())
-            ),
+        start_timestamp = start_timestamp or get_timestamp_from_utc_datetime(
+            get_current_datetime_in_utc()
         )
-        tick_times = []
+        upper_limit = upper_limit or 10
+        lower_limit = lower_limit or 10
```

Comment:
does defaulting 10 in the method decl not work? doesnt really matter though

Reply:
Not sure if it works but either way I think it is clearer if the sig matches the schema (i.e. the arg in the schema is nullable so it is good if the sig reflects that).